modules/cert-manager-configure-cloud-credentials-gcp-non-sts.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-authenticate-non-sts-gcp.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-prepare-cloud-credentials-gcp-non-sts"]
+= Configuring cloud credentials for the {cert-manager-operator} on GCP
+
+You can configure cloud credentials for the {cert-manager-operator} on GCP.
+
+.Prerequisites
+
+* You must extract and prepare the `ccoctl` binary.
```

Comment:
```suggestion
```
We can remove ccoctl dependency, as we do not need it for non-workload identity GCP.